lib/show_n_tell/user.ex

```diff
@@ -9,6 +9,8 @@ defmodule ShowNTell.User do
     field :github_token, :string
     field :last_name, :string
 
+    has_many :votes, ShowNTell.User
```

Comment:
In this case I don't think we will need to define a relation here. The reason I say that is because I don't think we will ever need to say "give me all the votes for this user". What do you think?

Reply:
I removed that line. I suppose we could force participation by banning users unless they have a high votes to talks ratio.  That can be implemented later..